Resources/Prototypes/Reagents/Consumable/Drink/alcohol.yml

```diff
@@ -1,13 +1,20 @@
 
 # Base Alcohol
 
+# TODO: drunkenness
+
 - type: reagent
   id: Absinthe
   name: absinthe
   desc: A anise-flavoured spirit derived from botanicals.
   physicalDesc: strong-smelling
   color: "#33EE00"
   spritePath: absintheglass.rsi
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
+        factor: 2
```

Comment:
Mein Gott is it not possible to use a parent prototype for these so you don't have to specify it every time?

Reply:
not for reagents yet

Reply:
would be nice and probably not too hard i can maybe look at how entities do it